test/integration/roles/test_ecs_ecr/tasks/main.yml

```diff
@@ -5,6 +5,10 @@
 - block:
     - name: When creating with check mode
       ecs_ecr: name='{{ ecr_name }}' region='{{ ec2_region }}'
+      environment:
```

Comment:
For tests that don't specifically need environment variables to make sure they're working correctly can we make these global to eliminate the repetition?